atlasdb-client/src/main/java/com/palantir/atlasdb/stream/AbstractPersistentStreamStore.java

```diff
@@ -45,6 +48,9 @@
 
 public abstract class AbstractPersistentStreamStore extends AbstractGenericStreamStore<Long>
         implements PersistentStreamStore {
+
+    public static final int NUM_RETRIES_ON_QOS_EXCEPTION = 6;
```

Comment:
For the first use case it may make sense to back off infinitely on RLE.  Certainly value in making this configurable.